src/rotation.zig

```diff
@@ -176,6 +177,39 @@ pub fn rotationMatrixTransform(quad: *main.models.QuadInfo, transformMatrix: Mat
 	}
 }
 
+fn rayTriangleIntersection(origin: Vec3f, direction: Vec3f, triangle: [3]Vec3f) ?f32 {
```

Comment:
If you use copy pasted code (which I assume you did here), then please link the source.